### ISSUE TITLE
White label customization

### ISSUE THREAD

**Issue by hugoattal** [histoire-dev/histoire#15]
Depends on #17 

We should have some kind of customization regarding the design (colors and font), and maybe the name/logo of the app.

- [x] `theme` config (below options are nested inside)
  - [x] `title`: name of the user project/company
  - [x] `favicon`
  - [x] `logoSquare`: square logo (favicon/home)
  - [x] `logoLight`: free-form logo for light mode
  - [x] `logoDark`: free-form logo for dark mode
  - [x] `primaryColors`: object for the primary color palette
  - [x] `grayColors`: object for the gray color palette
  - [x] ~`textFont`~
  - [x] ~`codeFont`~
- [x] Custom CSS (`setupFile`)
- [x] Auto generate CSS custom props in a standard file `__theme.css` and automatically include it in the app
- [x] Use those custom props in the histoire tailwind config for the `histoire` package (override)
- [ ] Add `750`, ` 850` and `950` shades to default gray colors (`zinc` already done) #35 

(Colors customization should also work with projects not using tailwind 😛)
